app/assets/stylesheets/elements/_dropdown.scss

```diff
@@ -124,6 +124,11 @@
     background-size: 70%;
   }
 }
+button.dropdown__item {
+  width: 100%;
+  background-color: transparent;
+  border: none;
+}
```

Comment:
Styling specific to an element is annoying because it's not predictable, I will merge my other PR first, and you should be able to remove this.